.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,29 @@
+name: Nightly
+on:
+  schedule:
+    - cron: '30 12 * * *'
+permissions:
+  id-token: write
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
```

Comment:
Since 1.18 came out, we may need to change this in the near future. Is it possible to query `go.mod` to see which version we're doing?

Reply:
that's something to change in multiple places not planning to include in this PR.

Reply:
done here https://github.com/aws/karpenter/pull/1578 good suggestion